NeuroID/src/main/java/com/neuroid/tracker/NeuroID.kt

```diff
@@ -519,4 +519,82 @@ class NeuroID private constructor(
     }
 
     fun getSDKVersion() = NIDVersion.getSDKVersion()
+
+    // new Session Commands
+    fun clearSessionVariables() {
+        userID = ""
+//        registeredUserID = ""
+    }
+
+    fun startSession(sessionID: String = ""): Pair<Boolean, String> {
```

Comment:
Instead of returning a pair, returning a class instance is better like a SessionResult `data class SessionResult(var result: Boolean, var sessionId: String)` or something like that.